pokemon-app/src/components/SinglePokemon.js

```diff
@@ -2,55 +2,94 @@ import fightPad from "../assets/fightpad.png";
+    <>
+      {!pokemon ? null : (
+        <>
+          <div
+            className="mainBackground "
+            style={{ backgroundImage: `url(${mainBackground})` }}
+          >
+            <div className="main-container">
+              <div
+                className={`Poke-display-container ${
+                  hasEnemy ? "has-enemy" : ""
+                }`}
+              >
+                <h1 className="poke-name">{pokemon.name}</h1>
+                <img
+                  className="poke-image"
+                  src={`${pokemon.sprites.front_default}`}
+                  alt="pokemon"
+                />
+              </div>
+
+              <div 
+                onClick={() => setStatsOnTop(true)}
+                className={
+                  `stats-title 
+                  ${hasEnemy ? "has-enemy" : ""} 
+                  ${statsOnTop ? "on-tip-top" : ""}`
+                }
+              >Stats:</div>
+              
+              <div 
+                onClick={() => setStatsOnTop(false)}
+                className={
+                  `moves-title 
+                  ${hasEnemy ? "has-enemy" : ""} 
+                  ${statsOnTop ? "" : "on-tip-top"}`
+                }
+              >Moves:</div>
+
+              <div className={`stats-box ${hasEnemy ? "has-enemy" : ""} ${statsOnTop ? "on-top" : ""}`}>
+                <ol id="stats-list">
+                  {!pokemon?.stats
+                    ? null
+                    : pokemon?.stats.map((statData, statIndex) => (
+                        <div key={statIndex}>
+                          <li key={`${statIndex}a`}>{statData.stat.name}:</li>
+                          <div key={`${statIndex}b`}>{statData.base_stat}</div>
+                        </div>
+                    ))
+                  }
+                </ol>
+              </div>
+
+              <div className={`moves-box ${hasEnemy ? "has-enemy" : ""} ${statsOnTop ? "" : "on-top"}`}>
+                <ol id="moves-list">
+                  {!pokemon?.moves
+                    ? null
+                    : pokemon?.moves.map((moveData, moveIndex) => (
+                        <li key={moveIndex}>{moveData.move.name}</li>
+                      ))
+                  }
+                </ol>
+              </div>
+
             </div>
+            <img
+              className={`poke-pad ${hasEnemy ? "has-enemy" : ""}`}
+              src={fightPad}
+              alt="lily pad from pokemon game"
+            />
           </div>
-          <img
-            className={`poke-pad ${hasEnemy ? "has-enemy" : ""}`}
-            src={fightPad}
-            alt="lily pad from pokemon game"
-          />
-        </div>
-      </>
-    )}
-
-    {!evolutions.length ? null : hasEnemy ? null : (
-      <PokemonEvolutions evolutions={evolutions} />
-    )}
-
-    {hasEnemy ? null : <TypeCheck pokemon={pokemon} />}
-  </>
-);
+        </>
+      )}
+
+      {!evolutions.length ? null : hasEnemy ? null : (
```

Comment:
You can simplify this to `(evolutions.length && hasEnemy) && ( <PokemonEvolutions evolutions={evolutions} /> )`

Reply:
https://reactjs.org/docs/conditional-rendering.html

Reply:
`true && <expression>` evaluates to `<expression>` in React.

Reply:
interesting. honestly never understood the && operator.

Reply:
Oooh, maybe I'll give a short lecture on boolean algebra.

Reply:
Hey if it give me more tools for my coding toolbelt then I'm all for it.

Reply:
But for now, when you see `this && that`, it means that the result of that boolean expression is true if and only if both sides of the `&&` operator evaluate to `true`.  For example:

```
true && true --> true
true && false --> false
false && true --> false
false && false --> false
```

Reply:
JS is weird, because `true && (<div>Hello</div>)` is `<div>Hello</div>`

Reply:
but thats just because theres nothing to say the right expressing if false right? You mean the fact that a div == true at all then yes lol

Reply:
![image](https://user-images.githubusercontent.com/9259239/126725979-49fd9b46-4c33-4d68-a9b8-3ce80cd7360f.png)